bundles/vi/spec/vi_spec.moon

```diff
@@ -105,17 +105,35 @@ describe 'VI', ->
   it '<t><character> searches forward in the current line to one character before <character>', ->
     press 't', 'n'
     assert.equal 2, cursor.column
+    press 't', 'q'
```

Comment:
I realize that this testes the cases where the cursor shouldn't move when no match is found, but that's not clear here just by reading this. Either a short comment here, or alternatively one could expand it into a block, e.g. 

```moonscript
describe '<t><character>', ->
  it 'searches forward[..]', ->
  it 'does nothing if <character> is not found', ->
```